proxies/nodejs/app.ts

```diff
@@ -1,20 +1,53 @@
-import { initialize } from '@devcycle/nodejs-server-sdk'
+import { DVCClient } from '@devcycle/nodejs-server-sdk'
 import Koa from 'koa'
+import Router from 'koa-router'
+import KoaBody  from 'koa-body'
 
-async function start() {
-    await initialize('<TOKEN>').onClientInitialized()
+const data = {
+  clients: {},
+  users: {},
+  commandResults: {}
+}
+
+const handleClient = async (ctx) => {
+  const body = ctx.request.body
+  if (body.clientId === undefined) {
+    ctx.status = 400 
+    ctx.body = "Invalid request: Missing clientId"
+  } else if (!body.sdkKey) {
+    ctx.status = 400 
+    ctx.body = "Invalid request: Missing sdkKey"
+  } else {
+    const client = new DVCClient(ctx.request.body.sdkKey, ctx.request.body.options)
```

Comment:
hmm, I think that the intention of the NodeJS SDK is that the end-user calls "initialize" rather than constructing the client directly like this. We should be calling whatever the method we expect the user to interact with here.

Also if possible we should probably prevent this style of creation of the client from being externally accessible, possibly by just exporting the type of DVCClient rather than the whole thing